slaves/blender/blender.ino

```diff
@@ -27,50 +31,70 @@ void setup() {
 
 void loop() {
     // Blender state machines
-    switch (states.blender1) {
+    switch (states.blender0) {
         case B_IDLE:
+            blender_control(BLEND_EN_0, LOW);
             break;
         case B_ON:
-            Serial.println("Blender 1 on");
+            blender_control(BLEND_EN_0, HIGH);
+            Serial.println("Blender 0 on");
             break;
-    };
-    switch (states.blender2) {
+    }
+    switch (states.blender1) {
         case B_IDLE:
+            blender_control(BLEND_EN_1, LOW);
             break;
         case B_ON:
-            Serial.println("Blender 2 on");
+            blender_control(BLEND_EN_1, HIGH);
+            Serial.println("Blender 1 on");
             break;
-    };
+    }
 
     // Pivot state machine
     switch (states.pivot) {
         case P_IDLE:
+            pivot_rotate(NEUTRAL, 0);
             break;
         case P_CW:
+            pivot_rotate(CW, PIVOT_SPEED);
             Serial.println("Turning CW");
             break;
         case P_CCW:
+            pivot_rotate(CCW, PIVOT_SPEED);
             Serial.println("Turning blender CCW");
             break;
         case P_HOME:
+            if (states.p_homed == 0) {
+                home_pivot();
+            } else {
+                pivot_setAngle(0);
+            }
             Serial.println("Homing pivot");
             break;
-    };
+        case P_ROTATE_180:
+            pivot_setAngle(180);
+            Serial.println("Rotating to 180");
+            break;
+    }
 
     // Elevator state machine
     switch (states.elevator) {
         case E_IDLE:
+            move_elevator(NEUTRAL, ELEV_SPEED);
```

Comment:
Might want to add a delay here and move to idle state. It's dangerous to indefinitely elevate or descend